test/embedding.jl

```diff
@@ -0,0 +1,12 @@
+# test that the embedding example runs without error
+let
+    if is_windows()
+        embedding_bin = joinpath(JULIA_HOME,"embedding.exe")
+    else
+        embedding_bin = joinpath(JULIA_HOME,"..","libexec","embedding")
```

Comment:
I don't think we need to install this at all (this PR currently does not install it). I'll try adding a variable for libexecdir tho.